ci/test/pipeline.template.yml

```diff
@@ -43,6 +43,20 @@ steps:
         agents:
           queue: builder-linux-x86_64
 
+      - id: rust-upload-debug-symbols-x86_64
+        label: ":rust: Upload debug symbols for x86_64"
+        env:
+          CI_BAZEL_BUILD: 0
+        command: bin/ci-builder run stable bin/pyactivate -m materialize.ci_util.upload_debug_symbols_to_polarsignals
+        inputs:
+          - "*"
+        depends_on: [rust-build-x86_64]
+        timeout_in_minutes: 10
+        priority: 50
+        if: build.tag != "" && build.branch =~ /^v.*\..*\$/
```

Comment:
At the moment, we can specify `BUILDKITE_TAG="true"` and it'll upload the debug symbols for a non-release branch build. The condition here seems like it requires the tag _and_ a release branch. If this is true, I'd change it to just requiring `build.tag ~= ""`.

Reply:
Yes, makes sense, I guess it should be a `||`.

Reply:
By the way, the original condition is currently used in two other places in this template file:
* `trigger: release-qualification`
* `trigger: nightly`

Reply:
✅ Changed my added entries.

Reply:
> ✅ Changed my added entries.

I reverted this because it cannot be correct if it triggers for this branch.